types/tx2/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for tx2 1.0
+// Project: https://github.com/pm2/tx2
+// Definitions by: Sasial <https://github.com/sasial-dev>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+import TX2 = require("./tx2");
```

Comment:
Out of curiosity, why the second required file here as opposed to just declaring everything here, which is a lot more common?

The actual source doesn't look like this, so it can't be used to simulate requiring specific files in the package or anything.

Reply:
The actual sorce does look like this - It's deciving!

I based it upon:
https://github.com/pm2/tx2/blob/main/index.js

Reply:
That's unfortunately not what's published; see the unpkg link in the first bot comment: https://unpkg.com/browse/tx2@1.0.5/

Reply:
Oh, no, it's the same, sorry. But, it's exporting `src`, not a module named `tx2`, so there's not much reason for the indirection if it doesn't match (or, nobody imports it like this), which is what I meant originally.

Reply:
What would you rather I do?
Just drop it in one file?